packages/markdown/remark/src/index.ts

```diff
@@ -39,6 +39,9 @@ export const markdownConfigDefaults: Omit<Required<AstroMarkdownOptions>, 'draft
 	smartypants: true,
 };
 
+// Skip nonessential plugins during performance benchmark runs
+const isPerformanceBenchmark = Boolean(process.env.ASTRO_PERFORMANCE_BENCHMARK);
```

Comment:
Don't _love_ this but if there's not a cleaner way to do it, that's fine.

Reply:
Yeah, couldn't come up with a clearer way. Originally had a private CLI flag with prop drilling, but wasn't sure that would be maintainable in the long run. Want to touch base with @bluwy about refactoring this performance package now that his benchmark package is in.

Reply:
Cool, resolving for now.